datahub-web-react/src/app/ingest/source/builder/RecipeForm/kafka.ts

```diff
@@ -0,0 +1,83 @@
+import { RecipeField, FieldType, setDottedFieldValuesOnRecipe } from './common';
+
+const saslUsernameFieldPath = ['source', 'config', 'connection', 'consumer_config.sasl.username'];
```

Comment:
`consumer_config` is going to be a parent, not the key for this field. so I'd remove that from this final string and make it come right before like:
`['source', 'config', 'connection', 'consumer_config', 'sasl.username']`

Reply:
Same thing goes for all fields below that have `consumer_config` in their fieldpath

Reply:
Done!